theme-switcher/switcher.js

```diff
@@ -0,0 +1,21 @@
+var themeList = document.getElementById('theme-list');
+
+function enableTheme(e) {
+  browser.management.setEnabled(e.target.value, true);
+  e.preventDefault();
+  window.close();
+}
+
+browser.management.getAll().then((themes) => {
+  for (let theme of themes) {
```

Comment:
I guess you mean https://bugzilla.mozilla.org/show_bug.cgi?id=1282981? Since it looks likely that this will be implemented, can we filter the results, as above, so this example keeps working when that lands?